packages/organizations/origin-organization-irec-api/src/irec/irec.module.ts

```diff
@@ -1,12 +1,29 @@
 import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { CommandBus, CqrsModule } from '@nestjs/cqrs';
+
 import { IrecService } from './irec.service';
-import { ConfigModule } from '@nestjs/config';
-import { CqrsModule } from '@nestjs/cqrs';
+import { IrecMockService } from './irec.mock.service';
+
+export const IREC_SERVICE = Symbol('IREC Service');
```

Comment:
Nice! But was it neccessary? Have you tried to use `IrecService` class as a provider token? Just asking

Reply:
Yep, it is
It does not work without this. It loads real IrecService when I tried to use class reference instead of Symbol

Reply:
Interesting... I will check this in free time